packages/cozy-harvest-lib/src/services/biWebView.js

```diff
@@ -322,6 +323,49 @@ function convertBIDateToStandardDate(biDate) {
   return biDate?.replace(' ', 'T')
 }
 
+async function getTokenFromCache({ client }) {
+  const queryResult = await client.query(
+    Q('io.cozy.bank.settings').getById('bi')
```

Comment:
Since this is specific to the process of creating a temporary token, then it should be named that way. `bi` is too generic. `temporaryBiToken` `biTmpToken` or else, but something specific please